Avoid installing variantconvert with pip because it is provided as an EasyBuild dependency.
Author: Emik Lin (HKUMed CPOS)
--- AnnotSV-3.5.10.orig/Makefile	2024-10-15 21:22:48.000000000 +0800
+++ AnnotSV-3.5.10/Makefile	2026-08-13 00:00:00.000000000 +0800
@@ -147,10 +147,6 @@ install-variantconvert:
 
 	touch $(VC_INSTALL_FLAG)
 	chmod 777 $(VC_INSTALL_FLAG)
-	pip3 install -e $(DESTDIR)$(PYTHON_INSTALL_DIR)/variantconvert/. > ./tmp.variantconvert.txt 2>&1 \
-	|| pip install -e $(DESTDIR)$(PYTHON_INSTALL_DIR)/variantconvert/. >> ./tmp.variantconvert.txt 2>&1 \
-	|| python -m pip install -e $(DESTDIR)$(PYTHON_INSTALL_DIR)/variantconvert/. >> ./tmp.variantconvert.txt 2>&1 \
-	|| rm -f $(VC_INSTALL_FLAG)
 	@if [ -f $(VC_INSTALL_FLAG) ]; then \
 		echo "variantconvert installed"; \
 		$(CHMOD) ./tmp.variantconvert.txt; \
